package.json

```diff
@@ -53,60 +53,60 @@
+    "@types/eslint": "^8.4.10",
+    "@types/express": "^4.17.14",
+    "@types/jsdom": "^20.0.1",
+    "@types/lodash": "^4.14.188",
+    "@types/mdast": "^3.0.10",
     "@types/node": "^18.11.9",
-    "@types/prettier": "2.7.1",
-    "@typescript-eslint/eslint-plugin": "5.42.1",
-    "@typescript-eslint/parser": "5.42.1",
-    "@vitest/coverage-c8": "0.25.1",
-    "@vitest/ui": "0.25.1",
-    "concurrently": "7.5.0",
-    "coveralls": "3.1.1",
-    "cypress": "10.11.0",
-    "cypress-image-snapshot": "4.0.1",
-    "esbuild": "0.15.13",
-    "eslint": "8.27.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-cypress": "2.12.1",
-    "eslint-plugin-html": "7.1.0",
-    "eslint-plugin-jest": "27.1.5",
-    "eslint-plugin-jsdoc": "39.6.2",
-    "eslint-plugin-json": "3.1.0",
-    "eslint-plugin-markdown": "3.0.0",
-    "eslint-plugin-no-only-tests": "3.1.0",
-    "eslint-plugin-tsdoc": "0.2.17",
-    "express": "4.18.2",
-    "globby": "13.1.2",
-    "husky": "8.0.2",
-    "identity-obj-proxy": "3.0.0",
-    "jest": "29.3.1",
-    "jison": "0.4.18",
-    "jsdom": "20.0.2",
-    "lint-staged": "13.0.3",
-    "path-browserify": "1.0.1",
-    "pnpm": "7.15.0",
-    "prettier": "2.7.1",
-    "prettier-plugin-jsdoc": "0.4.2",
-    "rimraf": "3.0.2",
-    "start-server-and-test": "1.14.0",
-    "ts-node": "10.9.1",
-    "typescript": "4.8.4",
-    "vite": "3.2.3",
-    "vitepress": "1.0.0-alpha.28",
-    "vitepress-plugin-mermaid": "2.0.8",
-    "vitepress-plugin-search": "1.0.4-alpha.15",
-    "vitest": "0.25.1"
+    "@types/prettier": "^2.7.1",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "@vitest/coverage-c8": "^0.25.1",
+    "@vitest/ui": "^0.25.1",
+    "concurrently": "^7.5.0",
+    "coveralls": "^3.1.1",
+    "cypress": "^10.11.0",
+    "cypress-image-snapshot": "^4.0.1",
+    "esbuild": "^0.15.13",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-html": "^7.1.0",
+    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-markdown": "^3.0.0",
+    "eslint-plugin-no-only-tests": "^3.1.0",
+    "eslint-plugin-tsdoc": "^0.2.17",
+    "express": "^4.18.2",
+    "globby": "^13.1.2",
+    "husky": "^8.0.2",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.3.1",
+    "jison": "^0.4.18",
+    "jsdom": "^20.0.2",
+    "lint-staged": "^13.0.3",
+    "path-browserify": "^1.0.1",
+    "pnpm": "^7.15.0",
+    "prettier": "^2.7.1",
+    "prettier-plugin-jsdoc": "^0.4.2",
+    "rimraf": "^3.0.2",
+    "start-server-and-test": "^1.14.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4",
+    "vite": "^3.2.3",
+    "vitepress": "^1.0.0-alpha.28",
+    "vitepress-plugin-mermaid": "^2.0.8",
+    "vitepress-plugin-search": "^1.0.4-alpha.15",
+    "vitest": "^0.25.1"
   },
   "resolutions": {
-    "d3": "7.6.1"
+    "d3": "^7.6.1"
```

Comment:
Knut mentioned there were issues with rendering pie charts with some d3 versions.
Can you please test if pie charts render properly?

Reply:
Yes, we should move on from abandoned packages.

- jison 
- dagre
- dagre-d3

3 of our core deps, all 3 deprecated. :(

Reply:
Pie charts still work with `"d3": "7.0.4"`, so I think it's fine to leave `"d3": "^7.0.0"` in mermaid's `dependencies`.

Again, that wouldn't change anything with the older version of `d3` found in `node_modules/dagre-d3/node_modules/d3`, which has `"d3": "^5.14`, which will happen when running `npm/yarn/pnpm install mermaid` (their package manager will ignore `mermaid`'s `"resolutions": {"d3": "^7.6.1"}` field).